app/helpers/controllers/manifestProvider.js

```diff
@@ -2,14 +2,14 @@ import fs from 'fs';
 import path from 'path';
 
 const orderItemTypeToFolderNameMap = {
-  solution: 'catalogue-solutions',
-  additionalservice: 'additional-services',
-  associatedservice: 'associated-services',
+  solution: 'catalogue-solutions/',
+  additionalservice: 'additional-services/',
+  associatedservice: 'associated-services/',
 };
 
 const getManifestPath = ({ orderItemType, provisioningType, type }) => {
   const orderItemsSectionPath = '../../pages/sections/order-items/';
-  const pathToManifest = `${orderItemTypeToFolderNameMap[orderItemType.toLowerCase()]}/order-item/${type.toLowerCase()}/${provisioningType.toLowerCase()}/manifest.json`;
+  const pathToManifest = `${orderItemTypeToFolderNameMap[orderItemType.toLowerCase()]}order-item/${type.toLowerCase()}/${provisioningType.toLowerCase()}/manifest.json`;
```

Comment:
Any particular reason for moving the the slash to the name map? I think it might have been slightly clearer as it was.

Reply:
I'd done a bigger change on that file and then reverted
will move the slash though yeah it's better in the top